server/views/pages/overview.njk

```diff
@@ -14,14 +16,19 @@
     {{ titleAndIntro(pathway) }}
       {{ tasksDetails(prisonerData.personalDetails.prisonerNumber, tasksViewEnabled) }}
   </div>
+ {% set releaseDateDays = prisonerData.personalDetails.releaseDate | getDaysFromDate %}
   {% if resettlementAssessmentEnabled and prisonerData.assessmentRequired %}
     {{ incompleteBCST2(pathway, prisonerData) }}
+    {% elif resettlementAssessmentEnabled and releaseDateDays.daysDiff <= 12 and prisonerData.resettlementReviewAvailable === true %}
```

Comment:
Should this be 12 weeks (84 days) instead of 12 days?

Reply:
Sorry, changed that now